R/board_connect_url.R

```diff
@@ -0,0 +1,154 @@
+#' Use a vector of Posit Connect vanity URLs as a board
+#'
+#' @description
+#' `board_connect_url()` lets you build up a board from individual
+#' [vanity urls](https://docs.posit.co/connect/user/content-settings/#custom-url).
+#' To use a Posit Connect board like this one or [board_connect()], you need to
+#' first authenticate. The easiest way to do so is by launching **Tools** -
+#' **Global Options** - **Publishing** - **Connect**, and follow the
+#' instructions.
+#'
+#' `board_connect_url()` is read only, and does not support either the use of a
+#' [manifest file][write_board_manifest()] or versioning.
```

Comment:
We can't use the approach to handle versioning that `board_url()` uses (no manifest files on Connect) or that `board_connect()` uses (no looking up GUIDs), so no versioning on this board.